tsconfig.json

```diff
@@ -1,34 +1,45 @@
- {
+{
+  "extends": "./tsconfig.paths.json",
   "compilerOptions": {
-    // copied from @tsconfig/node14/tsconfig.json (not working on ci)
-    "lib": ["ESNEXT"],
-    "module": "commonjs",
-    "target": "es2020",
-    "strict": true,
-    "esModuleInterop": true,
-    "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "outDir": "dist",
-    "baseUrl": ".",
-    "preserveConstEnums": true,
+
+        // copied from @tsconfig/node14/tsconfig.json (not working on ci)
+        "lib": ["ESNEXT"],
+        "module": "commonjs",
+        "target": "es2020",
+        "strict": true,
+        "esModuleInterop": true,
+        "skipLibCheck": true,
+        "forceConsistentCasingInFileNames": true,
+        "moduleResolution": "node",
+ //       "outDir": "dist",
+        "baseUrl": ".",
+        "preserveConstEnums": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "removeComments": true,
+        "sourceMap": true,
+        "outDir": "lib"
   },
   "parserOptions.projects": {
     "include": [
       "src/**/*"
     ],
   },
+  "include": ["src/**/*.ts", "serverless.ts", "index.ts", "index.ts", "src/functions/handler.ts", "src/functions/handler.js"],
```

Comment:
Why did you include these specific rules instead of the more general rules that were present before?

Reply:
this was from my problems before I had it working, changed it